src/styles/blocks/calendar.scss

```
@@ -0,0 +1,49 @@
+.calendar {
+  width: 799px;
+  min-height: 410px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: start;
+  padding: 10px;
+  box-sizing: border-box;
+  margin: auto;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $one-day-width;
+    height: 100px;
+    background-color: #eee;
+    border: 1px solid black;
+    box-sizing: border-box;
+    margin: $one-day-margin;
```

From this variable you have indent from all sides of square:
```suggestion
    gap: $one-day-margin;
    
```
gap-property helps you. Be careful - IE doesn't support gap-property

---

Не тільки internet explorer, а ще і Chromium, у якому запускаються тести. Але тут немає тестів, тому можна використовувати `gap` для флекса